src/bitmap/container.rs

```diff
@@ -38,13 +39,10 @@ impl Container {
         }
     }
 
-    pub fn insert_range(&mut self, range: Range<u16>) -> u64 {
-        // If the range is larger than the array limit, skip populating the
-        // array to then have to convert it to a bitmap anyway.
-        if matches!(self.store, Store::Array(_)) && range.end - range.start > ARRAY_LIMIT as u16 {
-            self.store = self.store.to_bitmap()
-        }
-
+    pub fn insert_range<R>(&mut self, range: R) -> u64
+    where
+        R: RangeBounds<u16>,
+    {
```

Comment:
As we are in the `Container` we should accept a `RangeInclusive` here.

Reply:
Infact, I think it's also need to use `RangeBounds` here since it's `pub` one. This may break the users who directly use this api. The same to the store.

Reply:
It is not public, as I told you in https://github.com/RoaringBitmap/roaring-rs/issues/113#issuecomment-925609379. Can we please continue speaking in only one place, it would be easier for me to follow the discussion, I would like that we exchange in this PR ;)